frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -768,6 +781,365 @@ class SegmentsView extends React.Component<Props, State> {
     </>
   );
 
+  getToastForMissingLocations = (groupId: number) => {
+    const segmentsWithoutPosition = this.getSegmentsWithMissingLocation(groupId);
+    if (segmentsWithoutPosition.length > 0) {
+      console.log(`Segments with unknown positions: ${segmentsWithoutPosition}`);
+      return Toast.info(
+        <React.Fragment>
+          {segmentsWithoutPosition.length} meshes couldn't be loaded because the segment position is
```

Comment:
this will read "1 meshes couldnt..." if length === 1. you can use the function `pluralize` for that.